tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -135,12 +135,24 @@ def test_requests_fail_if_no_status(self, svc, respond_with):
         with pytest.raises(LTIOutcomesAPIError):
             svc.read_result(self.GRADING_ID)
 
+    def test_requests_fail_if_response_is_malformed(self, svc, respond_with):
+        respond_with(malformed=True)
+
+        with pytest.raises(LTIOutcomesAPIError):
+            svc.read_result(self.GRADING_ID)
+
     def test_requests_fail_if_status_is_not_success(self, svc, respond_with):
         respond_with(status_code="failure")
 
         with pytest.raises(LTIOutcomesAPIError):
             svc.read_result(self.GRADING_ID)
 
+    def test_requests_fail_and_no_description_returned(self, svc, respond_with):
```

Comment:
Can you add a check to the above test to verify that the expected description is set on the response when a description is present.